src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames';
+import { memo } from 'react';
+import { Link } from 'react-router-dom';
+
+interface Props {
+  currentPath: string,
+}
```

Comment:
you could avoid passing pathname directly, instead, you'd rather apply `useParams` hook